prow/cluster/plank_deployment.yaml

```diff
@@ -19,13 +19,9 @@ spec:
         - --dry-run=false
         - --job-config-path=/etc/job-config
         - --tot-url=http://tot
-        - --github-endpoint=http://ghproxy
-        - --github-endpoint=https://api.github.com
         - --kubeconfig=/etc/kubeconfig/istio-config
+        - --skip-report=true
```

Comment:
Due to [this nonsense](https://github.com/kubernetes/test-infra/blob/ca8fda0be1599c631fcdba7d9a659f1dd0d5b8b4/prow/flagutil/github.go#L71-L75) you need to specify `--github-token-path=""`

Reply:
Are you certain this is needed? I noticed kubernetes/test-infra [plank deployment](https://github.com/kubernetes/test-infra/blob/ca8fda0be1599c631fcdba7d9a659f1dd0d5b8b4/prow/cluster/plank_deployment.yaml#L38-L45) does not explicitly set this either.

Reply:
Updated, I think I understand now.

Reply:
Yeah that would work. Alternatively we could switch plank to not default the token path (after a warning period). It does seem like `skip-report` and `--github-token-path` should be mutually exclusive though.